modules/administration-guide/partials/proc_installing-image-puller-using-che-operator.adoc

```diff
@@ -71,7 +71,7 @@ spec:
 $ {orch-cli} get clusterserviceversions
 ----
 
-. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-deployment-id}`.
+. The `eclipse-che namespace` contains these deployments: `{image-puller-deployment-name}` and `{image-puller-operator-repository-name}`.
```

Comment:
@boczkowska for me it is not really clear why deployment name is expected to be the repository name

Reply:
@ibuziuk I believe the deployment here is supposed to be `kubernetes-image-puller-operator`, we have two attributes that store this value:

>image-puller-operator-id: kubernetes-image-puller-operator
>image-puller-operator-repository-name: kubernetes-image-puller-operator

We don't have `image-puller-deployment-id` attribute defined at all. I mean, I could define it but is there a reason for us to store identical information in three attributes? The dowsntream docs aren't affected by these as far as I can see.

Reply:
@ibuziuk ...but thinking about it, you're right, naming an attribute properly and just defining it seems like a more straightforward way. Reverted this commit and defined the `image-puller-deployment-id` in antora playbook.

Reply:
Top-notch